crates/sargon/src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs

```diff
@@ -6,26 +6,34 @@ impl TransactionManifest {
     /// Creates the `ExecutionSummary` based on the `engine_toolkit_receipt` data.
     ///
     /// Such value should be obtained from the Gateway `/transaction/preview` endpoint, under the `radix_engine_toolkit_receipt` field.
-    /// The field will be a JSON that host apps should parse into a .
+    /// The field will be a JSON that host apps should parse into a transaction preview.
```

Comment:
Well, that's no longer the case. Now we are going to make the parsing of `radix_engine_toolkit_receipt` in Sargon.

Reply:
Actually, shouldn't this method be deleted now? We would only build the `ExecutionSummary` through SargonOS, which will internally call `execution_summary_with_engine_toolkit_receipt`

Reply:
Right!